doc/api/addons.md

```diff
@@ -311,7 +311,7 @@ it is compiled (i.e. sometimes it may be in `./build/Debug/`), Addons can use
 the [bindings][] package to load the compiled module.
 
 While the `bindings` package implementation is more sophisticated in how it
-locates Addon modules, it is essentially using a try-catch pattern similar to:
+locates Addon modules, it is essentially using a `try…catch` pattern similar to:
```

Comment:
Nit: should we use 3 dots instead of ellipsis character?

Reply:
No personal preference - I just copied what was already used in the errors docs.

Reply:
I'm also fine either way, just pointed out because 3 dots are used on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch

Reply:
OK, I've replaced the ellipsis throughout the docs and tests. I didn't update the CHANGELOGS though.

Reply:
Personal opinion: 3 dots are good for code, but ellipsis really do look nicer in prose text.

Reply:
> should we use 3 dots instead of ellipsis character?

> Personal opinion: 3 dots are good for code, but ellipsis really do look nicer in prose text.

I don't want to get hung up on this. I think I'll just drop the second commit from this PR.